### ISSUE TITLE
Typo in the section about Time Zones

### ISSUE THREAD

**Issue by jayarjo** [date-fns/date-fns#1249]
Writeup about the [data-fns-tz](https://date-fns.org/v2.0.0-alpha.26/docs/Time-Zones) has an example that starts like this:

```javascript
const { zonedTimeToUtc, utcToZonedTime, format } = require('date-fns-timezone')

// Set the date to "2018-09-01T16:01:36.386Z"
const utcDate = zonedTimeToUtc('2018-09-01 18:01:36.386', 'Europe/Berlin')
```

There's no `zonedTimeToUtc` function in `date-fns-timezone`, has to be:

```javascript
const { zonedTimeToUtc, utcToZonedTime, format } = require('date-fns-tz')
```

Confusing.

**Comment by diiorio** [comment]
The docs for the version you linked (`v2.0.0-alpha.26`) are outdated. The issue has been fixed since [`v2.0.0-alpha.37`](https://date-fns.org/v2.0.0-alpha.37/docs/Time-Zones). The latest version is currently [`v2.0.0-beta.2`](https://date-fns.org/v2.0.0-beta.2/docs/Time-Zones).

**Comment by kossnocorp** [comment]
@diiorio thank you for helping! 🤝
